cli/flox-rust-sdk/src/models/container_builder.rs

```diff
@@ -59,6 +59,17 @@ mod tests {
 
     use super::*;
 
+    /// OS error 26 is "Text file busy",
+    /// which can happen when executing a script
+    /// that is has been written to immediately before.
```

Comment:
nit: the race is caused by threading + forking

Reply:
I don't think its due to forking, at least not if this can be attributed to https://github.com/rust-lang/rust/issues/114554.

Even if `cargo test` runs by forking processes (which I'm not certain about either), each process will write to files in different tempdirs.

Reply:
`cargo test` is threading, `Command::new` is forking

Reply:
`Command::spawn == fork() -> exec()` _that_ fork, I see..

I think it's clear enough for now, for more detail there is the upstream issue linked as well